worksheets_test.go

```diff
@@ -84,6 +86,227 @@ func (s *Zuite) TestWorksheetNew_multipleDefsSameName() {
 	}
 }
 
+func (s *Zuite) TestExample_externalComputedBy() {
```

Comment:
Let's turn all these 'negative' tests into
```
cases := []struct{
  def string
  opt *Options
  expectedErr string
}{
  ...
}
for _, ex := range cases {
  var opts []Options
  if ex.opt != nil {
    opts = append(opts, opt)
  }
  err := NewDefinitions(strings.NewReader(ex.def), ...opts)
  ... etc.
}
```

applicable for `TestExample_externalComputedBy` to `TestExample_externalComputedBy4`